.github/workflows/on-pullrequest.yml

```diff
@@ -208,6 +207,24 @@ jobs:
           sudo systemctl start postgresql.service
           sudo -u postgres psql -U postgres -c "alter user postgres with password '${{env.POSTGRES_PASSWORD}}';"
 
+      - name: Create .env file
```

Comment:
With these changes, could we remove the e2e.env file for e2e tests and use this same step if we move to a common place? I think this is doing for integration tests what e2e.env did for e2e tests